konduit-serving-core/src/main/java/ai/konduit/serving/configprovider/PipelineRouteDefiner.java

```diff
@@ -172,8 +172,15 @@ public Router defineRoutes(Vertx vertx, InferenceConfiguration inferenceConfigur
         }
 
         healthCheckHandler = HealthCheckHandler.create(vertx);
+
+        /**
+         * Get a basic health check for a running Konduit server
```

Comment:
You can add this part here:

"If a server is up, this endpoint will return status of 204". Might be helpful for someone to figure out the output.

Reply:
done